microsite/src/main/tut/interop/catseffect.md

```diff
@@ -15,7 +15,7 @@ Checkout `interop-cats` module for inter-operation support.
 For convenience we have defined an alias as follow:
 
 ```scala
-  type Task[A] = Task[A]
+  type Task[A] = IO[Throwable, A]
```

Comment:
`scalaz.zio.Task[A]` .., or `IO` would be better? What do you think?

Reply:
What do you mean?
This line is just there to remind about what is Task. I guess it got replaced by itself in one of the renaming PRs.

Reply:
I saw interop-cats  I thought it was another type alias because in the code in interop there was Task type I was confused ^^'
Good job 👍